corehq/apps/analytics/tasks.py

```diff
@@ -178,7 +259,7 @@ def _send_post_data(url, params, data, headers):
 
 def _get_user_hubspot_id(webuser):
     api_key = settings.ANALYTICS_IDS.get('HUBSPOT_API_KEY', None)
-    if api_key and webuser.analytics_enabled:
+    if api_key and hubspot_enabled_for_user(webuser):
```

Comment:
Realize you didn't introduce this, but reading `webuser` instead of `web_user` is making me crazy.

Reply:
okay...i changed it for ya

Reply:
thank you!